File: Foundation/Scratch/Getting Started/c++/l002ReverseNumber.cpp
#include <iostream>
using namespace std;
void reverseNumber(int n){
    int ans=0,lastDigit=0;
    while(n){
        lastDigit = n%10;
        n /= 10;
         cout<<lastDigit<<endl;
       // ans = ans * 10 + lastDigit;
       
    }
    
}

int main(int argc, char **argv){
    int n;
    cin >> n;
reverseNumber(n);
    //write your code here
    return 0;
}

File: Foundation/Scratch/Getting Started/java/l002ReverseNumber.java
import java.util.*;

public class Main {
    public static Scanner scn = new Scanner(System.in);
public static void reverseNumber(int n){
    int ans=0,lastDigit=0;
    while(n!=0){
        lastDigit = n%10;
        n /= 10;
         System.out.println(lastDigit);
    }
    
}



    public static void main(String[] args) {
        // write your code here  
        int n = scn.nextInt();
        reverseNumber(n);
    }
}
